pkg/cmd/util/output/restore_describer.go

```diff
@@ -177,6 +177,11 @@ func DescribeRestore(ctx context.Context, kbClient kbclient.Client, restore *vel
 		d.Println()
 		d.Printf("Preserve Service NodePorts:\t%s\n", BoolPointerString(restore.Spec.PreserveNodePorts, "false", "true", "auto"))
 
+		if restore.Spec.UploaderConfigForRestore != nil && *restore.Spec.UploaderConfigForRestore.WriteSparseFiles {
```

Comment:
There is a risk of nil pointer b/c `restore.Spec.UploaderConfigForRestore.WriteSparseFiles`  may also be nil, you should use `boolptr.IsSetToTrue`

Reply:
updated